BREAKING.md

```diff
@@ -67,6 +62,18 @@ After:
 
 if for some reason this change causes you problems, we've added a deprecated `Loader._create` method that has the same parameters as the previous constructor which can be used in the interim.
 
+### Moving DriverHeader and merge with CreateNewHeader
+Compile time only API breaking change between runtime and driver.  Only impacts driver implementer.
+No back-compat or mix version impact.
+
+DriverHeader is a driver concept, so move from core-interface to driver-definitions. CreateNewHeader is also a kind of driver header, merged it into DriverHeader.
+
+### Added isRootDataStore property to IFluidDataStoreContext
```

Comment:
There is really nothing breaking about that (until we enable GC), so that note might be a bit confusing to people.
Maybe pre-phase it that's not breaking?

Reply:
isRootDataStore is not exposed anymore as per your other comment. So, removing this altogether from BREAKING.md